releasetasks/templates/final_verify.yml.tmpl

```diff
@@ -0,0 +1,49 @@
+{% set buildername = "{}_final_verify".format(branch) %}
+-
+    taskId: "{{ stableSlugId(buildername) }}"
+    reruns: 5
+    task:
+        provisionerId: aws-provisioner-v1
+        workerType: b2gtest
+        created: "{{ now }}"
+        deadline: "{{ now.replace(hours=6) }}"
```

Comment:
6 hours may not be enough, especially for releases, where this task will be able to run only after we "push to mirrors". I have a vague memory that TC has some limit for this (5 days?).